core/migrations/0029_alter_pumping_options_remove_pumping_time_and_more.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 4.2.2 on 2023-06-16 20:06
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def set_pumping_start_end_values(apps, schema_editor):
+    # The model must be imported to ensure its overridden `save` method is run.
+    from core.models import Pumping
```

Comment:
To ensure you have access to the old `time` field you'll need to get `Pumping` from `apps` -- e.g., `Pumping = apps.get_model("core", "Pumping")`. The `apps` variable represents the model _before_ the migration.

Reply:
fixed with [b7b28926c164a6cae38ff750fdbde79f1b2fa3e6](https://github.com/babybuddy/babybuddy/pull/665/commits/b7b28926c164a6cae38ff750fdbde79f1b2fa3e6)